src/gui/editors/PianoRoll.cpp

```diff
@@ -2118,9 +2118,14 @@ void PianoRoll::mouseMoveEvent( QMouseEvent * me )
 				bool isUnderPosition = n->withinRange( ticks_start, ticks_end );
 				// Play note under the cursor
 				if ( isUnderPosition ) { testPlayNote( n ); }
-				// If note is the one under the cursor or is selected when alt is
-				// not pressed
-				if ( ( isUnderPosition && !isSelection() ) || ( n->selected() && !altPressed ) )
+				// If note is:
+				// Under the cursor, when there is no selection
+				// Selected, and alt is not pressed
+				// Under the cursor, selected, and alt is pressed
+				if ( ( isUnderPosition && !isSelection() ) ||
+					  ( n->selected() && !altPressed ) ||
+					  ( isUnderPosition && n->selected() && altPressed )
+					)
```

Comment:
On Linux `<Alt>` + `<Left Mouse Button>` by default moves the whole application window. I think this is true on all desktops. This is nothing you've missed here but I had to change
`bool altPressed = me->modifiers() & Qt::AltModifier;` to 
`bool altPressed = me->modifiers() & Qt::ControlModifier;`
in order to test this. It works as intended on Linux with this change applied. My suggestion is to simply change `<Alt>` for `<Ctrl>` . Two issues fixed.